### ISSUE TITLE
Shadowlands WQ disapear

### ISSUE THREAD

**Issue by wKovacs** [leatrix/leatrix-maps#2]
Hello I like this littel add on but I have a minor issue.

I use in addition Classic Quest Log (https://www.curseforge.com/wow/addons/classic-quest-log). If I hit the Show Map all WorldQuest marker get lost and only a /reloadui fix it.

**Comment by leatrix** [comment]
When you click 'Show map' button, it shows the map for the selected quest so world quests don't show.  This happens whether Leatrix Maps is installed or not.  The world quests should show when you open the map with the M key though.  Do they?  If they do not, how often does this happen.  It works fine when I try it so is there a way for me to reproduce the issue?

**Comment by wKovacs** [comment]
Hello,

1.  I disabled ALL addons except Leatrix Maps and Classic Quest Log. 
2. hit <M> to view the map with WorldQuest markers
3. I open Classic Quest Log select one quest and hit Show Map Button on the top right
        (this shows the map without WQ markers)
4. Closing map and re-open again 
        (WQ markers still missing)
5. Right click on the map to show continent map or even world map will not trigger reload of WQ markers only /reloadui or relog is helping

Thanks for having a look into it

Regards
wKovacs
